pyvo/dal/params.py

```diff
@@ -387,16 +391,21 @@ def get_dal_format(self, val):
                                  format(val))
         else:
             high = low = val
+        if isinstance(low, (int, float)) and isinstance(high, (int, float))\
+                and low > high:
+            raise ValueError('Invalid interval: min({}) > max({})'.format(
+                low, high))
         if self._unit:
             if not isinstance(low, Quantity):
                 low = low*self._unit
             low = low.to(self._unit, equivalencies=self._equivalencies).value
             if not isinstance(high, Quantity):
                 high = high*self._unit
             high = high.to(self._unit, equivalencies=self._equivalencies).value
-        if low > high:
-            raise ValueError('Invalid interval: min({}) > max({})'.format(
-                low, high))
+
```

Comment:
@funbaker pardon for these unrelated changes I'm trying to get in with this PR. I need them for work in `astroquery.alma`

Reply:
Not a very clean practice to do unrelated changes in the same PR...
Not even a explanation what this is doing exactly?

This way it is not easy to revert, no one cane figure out why this is here, etc.

Reply:
Pull out. Now part of #225